samples/matter/thermostat/src/temperature_measurement/sensor.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include "binding_handler.h"
+#include <platform/CHIPDeviceLayer.h>
+#include <stdint.h>
+
+#include <atomic>
+
+#pragma once
+
+class MockedSensor {
+public:
+	static MockedSensor &Instance()
+	{
+		static MockedSensor sSensor;
+		return sSensor;
+	};
+
+	void TemperatureMeasurment();
+
+private:
+	int16_t previousTemperature;
+	uint8_t mockTempIndex;
+	uint8_t cycleCounter;
+};
+
+class RealSensor {
+public:
+	RealSensor();
+
+	static RealSensor &Instance()
+	{
+		static RealSensor sSensor;
+		return sSensor;
+	};
+
+	static void TemperatureMeasurment();
```

Comment:
```suggestion
	static void TemperatureMeasurement();
```
The same for `MockedSensor`